keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -152,18 +152,55 @@ def augment_image(self, image, transformation=None, **kwargs):
     def augment_label(self, label, transformation=None, **kwargs):
         return label
 
-    def augment_bounding_boxes(self, bounding_boxes, **kwargs):
-        # TODO: implements
-        return bounding_boxes
-
-    def augment_keypoints(self, keypoints, **kwargs):
-        # TODO: implements
-        return keypoints
+    def augment_bounding_boxes(
+        self, bounding_boxes, transformation, image=None, **kwargs
+    ):
+        if self.bounding_box_format is None:
+            raise ValueError(
+                "`RandomShear()` was called with bounding boxes,"
+                "but no `bounding_box_format` was specified in the constructor."
+                "Please specify a bounding box format in the constructor. i.e."
```

Comment:
looks like a bad merge occurred

Reply:
Strange there is no difference between `random_shear.py` master and c95017fd4d200c7f6575c98092fe079fb88e9864 (Current PR head). 
I just used the new version that only implements the bounding box augmentation

Reply:
> @atuleu lets perform a rebase!

@LukeWood  rebase Performed